bentoml/_internal/bento/build_config.py

```diff
@@ -380,10 +397,7 @@ class PythonOptions:
     )
     pip_args: t.Optional[str] = attr.field(
         default=None,
-        validator=attr.validators.optional(
-            lambda _, __, value: isinstance(value, str)
-            and re.match(r"^(?:--)\w*", value)
-        ),
+        validator=attr.validators.optional(attr.validators.instance_of(str)),
```

Comment:
I removed the regex check here since it doesn't add too much value and tricky to get it exactly right.

for future reference, we could be using something like this:

```python
        validator=attr.validators.optional([
            attr.validators.instance_of(str),
            attr.validators.matches_re(r"\s*(-{1,2}[^ =]+(=[^ =]+){0,1}[\s\n]*)+"),
        ]),
    )
```